docs/content/docs/dev/table/overview.md

```diff
@@ -40,8 +40,8 @@ and later use the DataStream API to build alerting based on the matched patterns
 
 ## Table Program Dependencies
 
-Depending on the target programming language, you need to add the Java or Scala API to a project
-in order to use the Table API & SQL for defining pipelines.
+Depending on the target programming language, you need to add the Table API to a project in order to
+use the Table API & SQL for defining data pipelines.
```

Comment:
I would rather go and remove these tabs all together. Perhaps just link to the project conf page for Java and Scala and the python overview page for Python? @MartijnVisser any opinions here?

Reply:
I would +1 removal of these tabs here. The overview page for Table should be comparable to the current DataStream Overview, so an overview of the Table APIs capabilities etc. We've moved the setup to the project configuration page, which fits better. Python is kind of a weird one with its own requirements, so for now better to link to the Python. 

So instead of having a section Table Program Dependencies, perhaps just a link to the project configuration page with a text like "Check out the project configuration for setting up your environment so you can start building a Table API program"? (But then in nicer phrasing 😅 )